src/oas/__tests__/__snapshots__/service.test.ts.snap

```diff
@@ -37,23 +37,17 @@ Object {
   ],
   "servers": Array [
     Object {
-      "description": undefined,
       "name": "Swagger Petstore",
       "url": "https://petstore.swagger.io/v2",
     },
     Object {
-      "description": undefined,
       "name": "Swagger Petstore",
       "url": "http://petstore.swagger.io/v2",
     },
   ],
   "tags": Array [
     Object {
       "description": "Everything about your Pets",
-      "externalDocs": Object {
```

Comment:
Do we not care about this anymore?

Reply:
It's not exposed on https://github.com/stoplightio/types/blob/master/src/graph.ts#L10, so I don't think it's needed.
The reason it was added in the first place is that we used to do something like https://github.com/stoplightio/http-spec/blob/master/src/oas3/service.ts#L115L118
In other words, we took the entire contents rather than cherry-picking what we actually want.
This isn't an isolated case, there were other scenarios where we did something like `omit(schema, keyA, keyB)` etc., which yet again isn't ideal because we may end up having plenty of junk if one has some, say, x-extensions in their document.
I went with allow-listing approach here rather than ignore-listing.